map/src/drawer/components/header/HeaderInfo.jsx

```diff
@@ -0,0 +1,139 @@
+import {Box, Typography} from "@mui/material";
+import React, {useContext, useEffect, useState} from "react";
+import AppContext, {toHHMMSS} from "../../../context/AppContext";
+import SearchInfo from "./SearchInfo";
+
+export default function HeaderInfo() {
+
+    const ctx = useContext(AppContext);
+
+    const [textInfo, setTextInfo] = useState({
+        tracks: {text: ''},
+        weather: {text: ''},
+        welcome: {text: ''}
+    });
+
+    useEffect(() => {
+        if (ctx.selectedObjects.tracks) {
```

Comment:
This code is unnecessary using extra state it could be just stored in app cotnext and here selected what to be displayed.

Reply:
`+`